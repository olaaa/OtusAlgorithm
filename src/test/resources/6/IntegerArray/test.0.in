12
94
10
91
4
3
31
22
95
87