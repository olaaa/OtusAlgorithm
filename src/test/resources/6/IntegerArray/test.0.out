[3, 4, 10, 12, 22, 31, 87, 91, 94, 95]